File: 35. Search Insert Position.js
let nums = [1,3,5,6], target = 5
let i;
for (i = 0; i < nums.length; i++ ) {
  console.log(i);
  if (nums[i] === target) {
    return i;
  }
  if (target < nums[i]) {
    return i;
  }
}
return i;

File: P/35_Search_Insert_Position.java
class Solution {
    public int searchInsert(int[] nums, int target) {
        int min = 0;
        int max = nums.length - 1;
        
        while(min <= max) {
            int mid = (min + max) / 2;
            if(nums[mid] == target)
                return mid;
            else if(nums[mid] < target)
                min = mid + 1;
            else
                max = mid - 1;
        }
        return min;
    }
}